pkg/operator/types.go

```diff
@@ -52,3 +60,10 @@ func MakeHostAliases(input []monitoringv1.HostAlias) []v1.HostAlias {
 
 	return output
 }
+
+func GetFieldJSONTag(parentField interface{}, fieldName string) string {
```

Comment:
parentField is a bit imprecise? It could also live in pkg/operator/rules.go if it's only used there.

```suggestion
func GetJSONTagForField(i interface{}, fieldName string) string {
```